config/locales/user_mailer/en.yml

```diff
@@ -224,6 +224,9 @@ en:
         sign_in_after_2fa: Authenticated
         sign_in_before_2fa: Signed in with password
         sign_in_unsuccessful_2fa: Failed to authenticate
+      failed_times:
+        one: ' %{count} time'
+        other: ' %{count} times'
```

Comment:
Just for my own edification: what was the motivation for putting the leading space here, instead of adding a space around ` %{failed_times}` in the `info_html` i18n key?

Reply:
The text is omitted altogether when there was a single failed attempt, so if it were in the other string, it could produce a result like...

>[...] failed to authenticate .

(extra space before the period)

I also thought about hard-coding a `' '` concatenation, though wasn't certain if that might cause us some trouble down the line with certain translations.

Reply:
I feel like the "right" thing to do there is move the count into that particular key then, because English lets us build a sentence piecemeal like that by removing words, but we don't know how accurate that translates to other languages, so more like:

```yaml
        info_html:
          - one: Your Login.gov email and password were used to sign in from a new
                 device but <strong>failed to authenticate</strong>.
           other: Your Login.gov email and password were used to sign in from a new
                  device but <strong>failed to authenticate %{count} times</strong>.

Reply:
and looking at the template, with the manual `I18n.interpolate` and manual `.html_safe` call... I think the array here is hurting more than it's helping, this would make for clearer code if we did

```
<p><%= t('info_p1_html', count: count) %></p>

<p><%= t('info_p2', count: count) %></p>

<p><%= t('info_p3_html', help_center_link_html: help_center_link_html) %></p>
```

Reply:
Yeah, that's a good point. Initially I expected broad consistency of the content between the templates, but that hasn't panned out. I also like the idea of embedding the distinction of "one" omitting "x times" into the full sentence, especially since it avoids having to figure out how to deal with the unused "1 time" (including translations).

I ended up refactoring this all in e934528 to split separate email templates and reference simplified hash key strings in the templates.